examples/arm/run_mcu_models_fvp.sh

```diff
@@ -0,0 +1,292 @@
+    echo ""
+
+    # Show individual results
+    for model in "${MODELS[@]}"; do
+        if [[ " ${PASSED_MODELS[*]} " =~ " $model " ]]; then
+            printf "%-12s : ✅ Passed\n" "$model"
+        elif [[ " ${FAILED_MODELS[*]} " =~ " $model " ]]; then
+            printf "%-12s : ❌ Failed\n" "$model"
+        else
+            printf "%-12s : ⏭️  Skipped\n" "$model"
+        fi
+    done
+
+    echo ""
+    echo "────────────────────────────────────────────────────────────────"
+
+    # Show statistics
+    local passed_count=${#PASSED_MODELS[@]}
+    local failed_count=${#FAILED_MODELS[@]}
+    local success_rate=$((passed_count * 100 / total_models))
+
+    echo "📊 STATISTICS:"
+    echo "   Total Models    : $total_models"
+    echo "   ✅ Passed       : $passed_count"
+    echo "   ❌ Failed       : $failed_count"
+    echo "   📈 Success Rate : $success_rate%"
+    echo ""
+
+    # Show model selection info
+    if [[ ${#MODELS[@]} -eq ${#DEFAULT_MODELS[@]} ]] && [[ "${MODELS[*]}" == "${DEFAULT_MODELS[*]}" ]]; then
+        echo "📋 Model Selection: Default (all available models)"
+    else
+        echo "📋 Model Selection: Custom (${MODELS[*]})"
+    fi
+    echo ""
+
+    # Overall result
+    if [[ $failed_count -eq 0 ]]; then
+        echo "🎉 OVERALL RESULT: ALL TESTS PASSED!"
+        echo "🔧 Mode: Portable Kernels (No Delegation)"
+    else
+        echo "⚠️  OVERALL RESULT: $failed_count/$total_models TESTS FAILED"
+        echo "🔧 Mode: Portable Kernels (No Delegation)"
+        echo ""
+        echo "🔍 Failed models: ${FAILED_MODELS[*]}"
+    fi
+
+    echo "════════════════════════════════════════════════════════════════"
+    echo ""
+}
+
+# Parse command line arguments
+while [[ $# -gt 0 ]]; do
+    case $1 in
+        --target=*)
+            TARGET="${1#*=}"
+            shift
+            ;;
+        --models=*)
+            IFS=',' read -ra MODELS <<< "${1#*=}"
+            shift
+            ;;
+        -h|--help)
+            show_usage
+            exit 0
+            ;;
+        *)
+            echo "❌ Error: Unknown argument '$1'"
+            echo ""
+            show_usage
+            exit 1
+            ;;
+    esac
+done
+
+# Check if target is provided
+if [[ -z "$TARGET" ]]; then
+    echo "❌ Error: --target argument is required"
+    echo ""
+    show_usage
+    exit 1
+fi
+
+# Validate target
+if ! validate_target "$TARGET"; then
+    echo "❌ Error: Invalid target '$TARGET'"
+    echo ""
+    show_usage
+    exit 1
+fi
+
+# Use default models if none specified
+if [[ ${#MODELS[@]} -eq 0 ]]; then
+    MODELS=("${DEFAULT_MODELS[@]}")
+fi
+
+# Validate models
+if ! validate_models; then
+    exit 1
+fi
+
+# Remove duplicates from models array
+IFS=" " read -r -a MODELS <<< "$(printf '%s\n' "${MODELS[@]}" | sort -u | tr '\n' ' ')"
+
+echo "🎯 MCU Model Validation - Target: $TARGET"
+echo "📋 Processing models: ${MODELS[*]}"
+echo "🔧 Mode: Portable Kernels (No Delegation)"
```

Comment:
We can have non-portable kernels, if we want to extend the "mode" in the future.